pkg/ccl/backupccl/restore_job.go

```diff
@@ -695,6 +716,7 @@ func createImportingDescriptors(
 	p sql.JobExecContext,
 	backupCodec keys.SQLCodec,
 	sqlDescs []catalog.Descriptor,
+	latestIntrosByIndex map[tableAndIndex]hlc.Timestamp,
```

Comment:
nit: maybe rename to `latestEndTimesForIntroducedSpans`

Reply:
renamed to latestEndTimesForIntroducedTables